src/module/item/spell/document.ts

```diff
@@ -152,19 +142,19 @@ class SpellPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Ite
     }
 
     get isAttack(): boolean {
-        return this.traits.has("attack") || this.system.spellType.value === "attack";
+        return this.traits.has("attack");
     }
 
     get isCantrip(): boolean {
         return this.traits.has("cantrip") && !this.isRitual;
     }
 
     get isFocusSpell(): boolean {
-        return this.system.category.value === "focus";
+        return this.traits.has("focus");
     }
 
     get isRitual(): boolean {
-        return this.system.category.value === "ritual";
+        return !!this.system.ritual;
```

Comment:
Is it possible in the new system for a spell to be a focus ritual? Should we relax some of the drag/drop restrictions in that case?